### PR TITLE
[WIP] first draft of adoptable modules

### PR DESCRIPTION
**Pull request by binford2k**
This is the first draft of a script to generate a list of modules up for
adoption. It will render as an HTML file and a JSON file.

Note that the page is not discoverable without an update to index.html
to link to it. I'll add that once this is reviewed and the cron action
is re-enabled.

See https://gist.github.com/binford2k/6dee4969a262a27535ace5e846b10a34 for example json output

### CODE REVIEW

**Comment by DavidS** [comment]
If you make it in the format of https://puppetlabs.github.io/iac/modules.json the list becomes consumable by other tooling we already have FWIW && YMMV.

File: adoptable_modules.html.erb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<html>
+<title>Puppetlabs modules up for adoption</title>
+<head>
```
Comment (DavidS) [inline adoptable_modules.html.erb:3]: please add the GA tracker like in https://github.com/puppetlabs/community_management/blob/a537bf8c14c011537687c795678f00577a971798/index.html#L9-L20

Comment (DavidS) [reply adoptable_modules.html.erb:3]: looking through the other reports, we need to add it to a couple of pages.

**Comment by DavidS** [comment]
apologies, didn't have a fresh head for a full review - if the output is correct, the coding isn't that important anyways.

**Comment by binford2k** [comment]
> If you make it in the format of https://puppetlabs.github.io/iac/modules.json the list becomes consumable by other tooling we already have FWIW && YMMV.

@DavidS is that format documented somewhere? Most of the fields do not appear relevant here. What needs to be implemented for consumption by other tooling?

**Review by michaeltlombardi — APPROVED** [review]
Looks good to me!

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**